pom.xml

```diff
@@ -49,7 +49,7 @@
       <tag>HEAD</tag>
   </scm>
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jenkins.version>1.609.2</jenkins.version>
```

Comment:
Ok, it was done to [pick up JENKINS-28840](https://github.com/jenkinsci/workflow-plugin/pull/184) to avoid failing tests, but in that moment 1.609.2 was just released and forcing a core update just for testing purposes was avoided (thus #197).

I guess the override could be removed now (there is even a new LTS line out there).